Gemfile.lock

```diff
@@ -0,0 +1,329 @@
+GEM
```

Comment:
I'd avoid adding this for a GitHub Pages repository as we're not actually running these versions, just removed it and have CI failures that can happen due to external factors already.